src/components/Quote/Template/index.tsx

```diff
@@ -0,0 +1,204 @@
+      <Typography variant="body1">Quote Chat</Typography>
+      <Typography variant="h2" sx={{ color: '#1E6FFF', mb: 2 }}>
+        {showNegotiate ? 'Negotiate Quote' : 'Merchant Quote'}
+      </Typography>
+
+      <Divider />
+
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: showNegotiate
+            ? 'repeat(4, 1fr)'
+            : 'repeat(3, 1fr)',
+          gap: 2,
+          position: 'relative',
+        }}
+      >
+        <Typography variant="body1" fontWeight="bold">
+          Files
+        </Typography>
+        <Divider
+          sx={{ position: 'absolute', left: showNegotiate ? '25%' : '33%' }}
+          orientation="vertical"
+        />
+        <Typography variant="body1" fontWeight="bold" textAlign="center">
+          Quantity
+        </Typography>
+        <Divider
+          sx={{ position: 'absolute', left: showNegotiate ? '50%' : '66%' }}
+          orientation="vertical"
+        />
+        <Typography
+          variant="body1"
+          fontWeight="bold"
+          textAlign={showNegotiate ? 'center' : 'right'}
+        >
+          Pricing
+        </Typography>
+        {showNegotiate && (
+          <Divider
+            sx={{ position: 'absolute', left: '75%' }}
+            orientation="vertical"
+          />
+        )}
+        {showNegotiate && (
+          <Typography variant="body1" fontWeight="bold" color="#1E6FFF">
+            Price Negotiate
+          </Typography>
+        )}
+
+        {quote.Items.map((item, index) => (
+          <React.Fragment key={item.file}>
+            <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+              <Box
+                sx={{
+                  height: 32,
+                  width: 32,
+                  backgroundColor: '#DDE9FC',
+                  borderRadius: '25%',
+                }}
+              />
+              <Typography variant="body2">{item.file}</Typography>
+            </Box>
+            <Typography variant="body2" textAlign="center">
+              {item.quantity}
+            </Typography>
+            <Typography
+              variant="body2"
+              textAlign={showNegotiate ? 'center' : 'right'}
+            >
+              ${item.price.toFixed(2)}
+            </Typography>
+            {showNegotiate && (
+              <TextField
+                size="small"
+                value={item.price}
+                onChange={(e) => handlePriceChange(index, e.target.value)}
+                sx={{ '& .MuiOutlinedInput-root': { bgcolor: '#DDE9FC' } }}
+              />
+            )}
+          </React.Fragment>
+        ))}
+      </Box>
+
+      <Divider sx={{ mb: 2 }} />
+
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: showNegotiate
+            ? 'repeat(4, 1fr)'
+            : 'repeat(3, 1fr)',
+          gap: 2,
+          mb: 2,
+        }}
+      >
+        <Typography variant="body2">Total Price</Typography>
```

Comment:
can you use maps to write all this?? seems like the code is repeating alot with few constants changes